app/src/modules/settings/routes/data-model/field-detail/field-detail-advanced/field-detail-advanced-schema.vue

```diff
@@ -444,7 +445,12 @@ function useOnUpdate() {
 
 			<div v-if="!isAlias" class="field half-right">
 				<div class="label type-label">{{ t('unique') }}</div>
-				<v-checkbox v-model="unique" :disabled="isGenerated" :label="t('value_unique')" block />
+				<v-checkbox v-model="unique" :disabled="isGenerated || indexed" :label="t('value_unique')" block />
+			</div>
+
+			<div v-if="!isAlias" class="field half-left">
+				<div class="label type-label">{{ t('indexed') }}</div>
+				<v-checkbox v-model="indexed" :disabled="unique || isGenerated" :label="t('value_indexed')" block />
```

Comment:
does it make sense to allow both unique and index for a column? i feel its redundant

Reply:
in fact oracle doesnt allow both unique and normal index on a column.

Reply:
We should not restrict unique + index for all drivers due to one driver. We should allow this for all drivers, if the driver does not support both (i.e. oracle) then if one is active and the other is selected then it should be ignored but the API should return both as active.

Reply:
ok thats a good suggestion, thanks. i will return both values if one of them is true.

Reply:
If it is possible it is better if both are returned as true only if the index was selected as well.

e.g.
- unique or index selected -> all drivers return only the respective field as true
- unique + index are selected
   - driver supports both -> both returned as true and set at db level
   - driver does not support both -> both returned as true but only one of them (which ever is selected first) are enforced at the db level.

Reply:
> unique or index selected -> all drivers return only the respective field as true

in this case users might select the other checkbox, but always get error on unsupported driver. 

> unique + index are selected
driver supports both -> both returned as true and set at db level

we could do that, but that would be redundant as unique itself is a superset of simple index. 

imo, it makes sense to always return true for `index` if it is `unique`, but returning unique=true when its indexed would give wrong info. so we have to handle it at UI side or users will select it and either nothing happens(on backend i already ignore `index` request when field is already unique), or we throw an error .